scripts/auto-stop-idle/on-start.sh

```diff
@@ -24,10 +24,10 @@ echo "Detecting Python install with boto3 install"
 
 # Find which install has boto3 and use that to run the cron command. So will use default when available
 # Redirect stderr as it is unneeded
-if /usr/bin/python3 -c "import boto3" 2>/dev/null; then
+if /usr/bin/python3.8 -c "import boto3" 2>/dev/null; then
+    PYTHON_DIR='/usr/bin/python3.8'
+elif /usr/bin/python3 -c "import boto3" 2>/dev/null; then
     PYTHON_DIR='/usr/bin/python3'
-elif /usr/bin/python -c "import boto3" 2>/dev/null; then
-    PYTHON_DIR='/usr/bin/python'
 else
```

Comment:
Not a very scalable solution. I'm wondering if this script can use the python library in JupyterSystemEnv.

Reply:
Resolved in new commits to use $(which python) instead.